docker-build/README.md

```diff
@@ -0,0 +1,31 @@
+# Docker build 
+
+For a cleanish and immutable build 
+
+Typically run like this for a local build
+
+`% DOCKER_BUILDKIT=1 docker build -t nlc-drupal -f docker-build/Dockerfile .`
+
+(note - the build expects a version file as build-version.txt in the root which gets added by
+concourse - this will need manually adding for the local build. it's just informational so touch
+the file there)
+
+*As of current docker version (19.03) the DOCKER_BUILDKIT is used to allow per build docker ignore files 
+
+The overall process is as follows 
+
+- Something runs composer (either a developer locally or else a concourse pipeline) for PHP installs
+(* and then ny npm installs - TODO) to get the checked out version fully setup, so as this build can create an
```

Comment:
Right, yes, deets on the necessary `npm install` process noted. Will pass that across to you.